### PR TITLE
Fix image and title

### PR DESCRIPTION
**Pull request by Hoverbear**
Fixes the title and a bad image link.

### CODE REVIEW

File: src/overview/introduction.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Fortunately, Google met the same problem and had already published some papers t
 
 Following is the architecture of TiKV:
 
-![Architecture](architecture.png)
+![Architecture](../overview/architecture.png)
```
Comment (siddontang) [inline src/overview/introduction.md:27]: oh, strange 😭 

can `./architecture.png` work?

Comment (MyonKeminta) [reply src/overview/introduction.md:27]: Seems not. It tries to find "/deep-dive-tikv/book/architecture.png".

**Review by siddontang — APPROVED** [review]
LGTM

**Comment by MyonKeminta** [comment]
I tried changing the image link this way on https://tikv.github.io/deep-dive-tikv/book/ . Unfortunately, it tries to find https://tikv.github.io/deep-dive-tikv/overview/architecture.png while the actual location is https://tikv.github.io/deep-dive-tikv/book/overview/architecture.png . This PR didn't solve the Problem.
@siddontang

**Comment by siddontang** [comment]
PTAL @Hoverbear

File: index.html [hunk 2]
```diff
@@ -1 +1 @@
-<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=book/"></head><body></body></html>
+<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=book/overview/introduction.html"></head><body></body></html>
```
Comment (Hoverbear) [inline index.html:1]: cc @MyonKeminta See this line.

File: src/overview/introduction.md (hunk 1 above)
Comment (Hoverbear) [reply src/overview/introduction.md:27]: Only for this image. All others do not need this. It's only because it's a special case, the first page.

**Comment by Hoverbear** [comment]
@MyonKeminta This PR changes the landing of **https://tikv.github.io/deep-dive-tikv/** so that it is https://tikv.github.io/deep-dive-tikv/book/overview/introduction.html instead of https://tikv.github.io/deep-dive-tikv/book/.

It is the only way to fix this both for local builds and published builds it seems

File: index.html (hunk 2 above)
Comment (MyonKeminta) [reply index.html:1]: Ouch, sorry i didn't notice that.

**Review by MyonKeminta — APPROVED** [review]
LGTM

File: index.html (hunk 2 above)
Comment (Hoverbear) [reply index.html:1]: Don't be! :) Very happy to have you look at my work.

**Pull request merged**